contrib/checkbox-ce-oem/checkbox-provider-ce-oem/data/video-codec-test-confs/genio-350.json

```diff
@@ -0,0 +1,14 @@
+{
+    "gst_v4l2_audio_video_synchronization": {
+        "video_sinks": {
+            "non_desktop": "kmssink connector-id=32 driver-name=mediatek"
+        },
```

Comment:
Since G350 platform doesn't support Desktop image, therefore, we can safely ignore the waylandsink here